ewatercycle/models/abstract.py

```diff
@@ -42,6 +42,16 @@ def __del__(self):
         except AttributeError:
             pass
 
+    def __str__(self):
+        """Nice formatting of model object."""
+        return "\n".join(
+            [
+                f"eWaterCycle {self.__class__.__name__}",
+                "-------------------",
+            ]
+            + [f"{k}={v!s}" for k, v in self.__dict__.items()]
```

Comment:
make this only print the name of the model, and the name of the parameterset and forcing.
```suggestion
            + [f"Parameter set = {self.parameter_set}]
            + [f"Forcing = {self.forcing}]
            + [f"Version = {self.version}]
```